projects/components/select/src/select.component.html

```diff
@@ -0,0 +1,53 @@
+<div [matTooltip]="tooltip" [matTooltipDisabled]="!multiple">
+  <mat-select
+    [value]="value"
+    [disableOptionCentering]="true"
+    [compareWith]="compareWith"
+    [disabled]="disabled"
+    [multiple]="multiple"
+    [errorStateMatcher]="errorStateMatcher"
+    [panelClass]="panelClass"
+    [placeholder]="placeholder"
+    [required]="required"
+    (valueChange)="onValueChange($event)"
+    (selectionChange)="onSelectionChange($event)"
+    (openedChange)="onOpenedChange($event)"
+  >
+    <mat-select-trigger *ngIf="triggerTemplate && !empty">
+      <ng-container [ngTemplateOutlet]="triggerTemplate" [ngTemplateOutletContext]="{ $implicit: customTriggerData }"></ng-container>
+    </mat-select-trigger>
+
+    <mat-option class="ps-select__search">
+      <ngx-mat-select-search
+        [formControl]="filterCtrl"
+        [searching]="loading"
+        [showToggleAllCheckbox]="showToggleAll"
+        [toggleAllCheckboxChecked]="toggleAllCheckboxChecked"
+        [toggleAllCheckboxIndeterminate]="toggleAllCheckboxIndeterminate"
+        (toggleAll)="onToggleAll($event)"
+      ></ngx-mat-select-search>
+    </mat-option>
+
+    <mat-option *ngIf="hasError" [disabled]="true" class="ps-select__error">
```

Comment:
There is no definition for this class in the scss

Reply:
it's a marker class